readme.md

```diff
@@ -13,31 +13,70 @@ npm install @cycle/react-native
 
 ```js
 import xs from 'xstream';
-import {run} from '@cycle/run';
-import {makeReactNativeDriver, TouchableOpacity, View, Text} from '@cycle/react-native';
+import {setup} from '@cycle/run';
+import {
+  makeReactNativeDriver,
+  View,
+  TouchableHighlight,
+  Text,
+} from '@cycle/react-native';
+import {name as appName} from './app.json';
+
+const styles = {
+  view: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100%',
+  },
+  button: {
+    backgroundColor: '#191815',
+    borderWidth: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: 10,
+    width: 150,
+    borderRadius: 10,
+  },
+  buttonText: {
+    color: 'white',
+  },
+};
 
 function main(sources) {
   const inc = Symbol();
-  const inc$ = sources.react.select(inc).events('click');
+  const inc$ = sources.react.select(inc).events('press');
 
   const count$ = inc$.fold(count => count + 1, 0);
 
   const elem$ = count$.map(i =>
-    TouchableOpacity(inc, [
-      View([
-        Text(`Counter: ${i}`),
-      ])
-    ]),
+    View(
+      {
+        style: styles.view,
+      },
+      [
+        TouchableHighlight(
+          inc,
+          {
+            style: styles.button,
+          },
+          [View([Text({style: styles.buttonText}, `Counter: ${i}`)])],
+        ),
+      ],
+    ),
   );
 
   return {
     react: elem$,
   };
 }
 
-run(main, {
```

Comment:
Did `run` not work? It should work.

Reply:
Nope, was left out at a blank render unless a delay was added

Reply:
Then that's a bug, `run` needs to work (I'm sure it once worked).

Reply:
Then that's a bug, `run` needs to work (I'm sure it once worked).

Reply:
In that case the PR can be ignored, I'll try to look into the run issue